src/components/CreateInstructorPage/CreateInstructorForm.test.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { extractOrgChoices, BaseCreateInstructorForm, basicValidate } from './CreateInstructorForm';
+
+
+const instructorOptions = {
+  data: {
+    actions: {
+      POST: {
+        position: {
+          children: {
+            organization: {
+              choices: [
+                { display_name: 'edX', value: 1 },
+                { display_name: 'bananasX', value: 2 },
+              ],
+            },
+          },
+        },
+      },
+    },
+  },
+};
+
+describe('CreateInstructorForm', () => {
+  it('renders html correctly', () => {
+    const component = shallow(<BaseCreateInstructorForm
```

Comment:
I think all of publisher could benefit from snapshot testing base components rather than connected components... but that's probably low priority